### PR TITLE
[TW-774] Wording changes to CI integrations

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Updated `To view a build...` to `To open a build...` because "open" clarifies that the user will open the build in the CI tool, not in Postman.
* Clarified in the "Configuring the Postman CLI" section that Postman CLI runs Postman collections with API tests, not API tests.

### CODE REVIEW

File: src/pages/docs/integrations/available-integrations/ci-integrations/bitbucket-pipelines.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ To view the results of API Governance and API Security checks that ran as part o
 
 ## Configuring the Postman CLI for Bitbucket Pipelines
 
-With the help of the Postman CLI and the Postman API, you can run API tests created in Postman as part of your Bitbucket pipeline. First generate the Postman CLI configuration code in Postman. Then add the configuration code to the `bitbucket-pipelines.yml` file in your Bitbucket repository.
+With the help of the Postman CLI and the Postman API, you can run Postman collections with your API tests created in Postman as part of your Bitbucket pipeline. First generate the Postman CLI configuration code in Postman. Then add the configuration code to the `bitbucket-pipelines.yml` file in your Bitbucket repository.
```
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/integrations/available-integrations/ci-integrations/bitbucket-pipelines.md:72]: I like the change to "Postman collections with your API tests" but I think it makes "created in Postman" redundant 😄 Same for the occurrences on other pages

**Review by mark-anthony-postman — APPROVED** [review]
Looks good! 🌟 Made one suggestion on wording on you consider

File: src/pages/docs/integrations/available-integrations/ci-integrations/bitbucket-pipelines.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MatthewPinsker-postman) [reply src/pages/docs/integrations/available-integrations/ci-integrations/bitbucket-pipelines.md:72]: Yes, I agree. Thanks!

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**